### PR TITLE
Add periodic test for devfile

### PR DESCRIPTION
**Pull request by josephca**
Add periodic test run for devfile integration test

### CODE REVIEW

File: ci-operator/config/devfile/integration-tests/devfile-integration-tests-master__v4.5.yaml [hunk 1]
```diff
@@ -30,7 +30,27 @@ tests:
...
     - as: integration-e2e-steps
-      commands: scripts/openshiftci-presubmit-all-tests.sh
+      commands: scripts/openshiftci-presubmit-devfiles-tests.sh
+      credentials:
+      - mount_path: /usr/local/ci-secrets/odo-rabbitmq
+        name: odo-rabbitmq
+        namespace: test-credentials
+      env:
+      - default: /usr/local/ci-secrets/odo-rabbitmq/amqpuri
+        name: ODO_RABBITMQ_AMQP_URL
```
Comment (amitkrout) [inline ci-operator/config/devfile/integration-tests/devfile-integration-tests-master__v4.5.yaml:40]: @josephca These credentials are not require from devfile repo point view. BTW you are not mapping any credentials for devfile integration repo.

Please remove these credentials and env fields.

**Review by amitkrout — CHANGES_REQUESTED** [review]

**Review by amitkrout — CHANGES_REQUESTED** [review]
use latest released 4.6 cluster. For example - https://github.com/openshift/release/blob/master/ci-operator/config/openshift/odo/openshift-odo-master__v4.6.yaml

File: ci-operator/config/devfile/integration-tests/devfile-integration-tests-master__v4.5.yaml [hunk 2]
```diff
@@ -30,7 +30,27 @@ tests:
     cluster_profile: aws
     test:
     - as: integration-e2e-steps
```
Comment (amitkrout) [inline ci-operator/config/devfile/integration-tests/devfile-integration-tests-master__v4.5.yaml:32]: integration-e2e-steps -> integration-devfile-odo-steps

Also change https://github.com/openshift/release/pull/15184/files#diff-28ce97f152f3eef114da77fb175110b57ca18c74e17462b53e9891f8383dd39eR28 to integration-devfile-odo

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: ci-operator/config/devfile/integration-tests/devfile-integration-tests-master__v4.5.yaml [hunk 3]
```diff
@@ -30,7 +30,27 @@ tests:
...
+      env:
+      - default: /usr/local/ci-secrets/odo-rabbitmq/amqpuri
+        name: ODO_RABBITMQ_AMQP_URL
+      from: oc-bin-image
+      resources:
+        requests:
+          cpu: "2"
+          memory: 6Gi
+    workflow: ipi-aws
+- as: integration-e2e-periodic
```
Comment (amitkrout) [inline ci-operator/config/devfile/integration-tests/devfile-integration-tests-master__v4.5.yaml:47]: integration-e2e-periodic -> integration-devfile-odo-periodic

File: ci-operator/config/devfile/integration-tests/devfile-integration-tests-master__v4.5.yaml [hunk 4]
```diff
@@ -30,7 +30,27 @@ tests:
...
+  cron: 0 */6 * * *
+  steps:
+    cluster_profile: aws
+    test:
+    - as: integration-e2e-periodic-steps
```
Comment (amitkrout) [inline ci-operator/config/devfile/integration-tests/devfile-integration-tests-master__v4.5.yaml:52]: integration-e2e-periodic-steps -> integration-devfile-odo-periodic-steps

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml [hunk 5]
```diff
@@ -22,6 +22,7 @@ postsubmits:
         - --report-password-file=/etc/report/password.txt
         - --report-username=ci
         - --target=[images]
+        - --upload-via-pod-utils
```
Comment (amitkrout) [inline ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:25]: @josephca I am not sure about this change. Can you please rebase the pr

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml (hunk 5 above)
Comment (josephca) [reply ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:25]: @amitkrout Actually, this change was generated by running 'make jobs' and this change is already available in 'master' branch, thus the merge would ignore this change. Nonetheless, in order to keep the commit history clean, I created a new PR by rebasing it to master again - https://github.com/openshift/release/pull/15267

**Comment by josephca** [comment]
This PR is replaced by https://github.com/openshift/release/pull/15267 which contains the rebase and will result in the clean linear commit history. Closing it with no change.

**Pull request closed without merging**